controllers/historyController.js

```diff
@@ -19,8 +19,8 @@ class HistoryController {
 				if (!err) {
 					testList = data;
 				}
-				const location = testList[0] ? testList[0].location : 'N/A';
-				callback(null, {testLocation: location, testList: testList, testName: testName});
+
+				callback(null, {testURL: testList[0] && testList[0].url, testList: testList, testName: testName});
```

Comment:
Use ES6 [shorthand syntax](http://ariya.ofilabs.com/2013/02/es6-and-object-literal-property-value-shorthand.html).